migrations/2_deploy_contracts.js

```diff
@@ -0,0 +1,14 @@
+const MultiSigWallet = artifacts.require("./gnosis/MultiSigWallet.sol");
+const TokenAuthority = artifacts.require("./TokenAuthority.sol");
+const Token = artifacts.require("./Token.sol");
+const Vesting = artifacts.require("./Vesting.sol");
+
+module.exports = (deployer, network, accounts) => {
```

Comment:
Because it's just standard arguments (not destructured from JSON) I think the order matters, so we need to keep `network` in order to access `accounts`.

Reply:
Sure, if I remove the `network` argument, when I `console.log(accounts)` (now the second argument) it outputs `development`.

Reply:
No mention of this in documentation http://truffleframework.com/docs/getting_started/migrations I'll check with them on gitter

Reply:
But that shouldn't stop this PR. It's just interesting to find out